src/AppInstallerCLICore/Core.cpp

```diff
@@ -61,7 +63,7 @@ namespace AppInstaller::CLI
         // Initiate the background cleanup of the log file location.
         Logging::BeginLogFileCleanup();
 
-        Execution::Context context{ std::cout, std::cin };
+        Execution::Context context{ std::cout, std::cin, nullptr /* Command execution always use global diagnostic logger */, Logging::Telemetry().CreateSubTraceLogger() };
```

Comment:
If we are moving to the CLI using context specific objects, we should go all the way and not ever use the global one.  I would prefer that we didn't have a global one anyway, but getting rid of it was too difficult at the time.

I can see the desire to have a single file logger, but I think that the telemetry should be rooted here, not a child of the global telemetry.

And then set this as the active context of course.